Cargo.lock

```diff
@@ -4735,18 +4720,19 @@ dependencies = [
 
 [[package]]
 name = "soroban-sdk"
-version = "22.0.0-rc.1.1"
+version = "22.0.0-rc.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "68b715e15357dbdda2fe0626005b75c2253bb3786deec3001d4077462a9dadd3"
+checksum = "6d063d0df000aaec20105aab3d743660322bc0269934ea95d79fa19aa8792385"
 dependencies = [
  "bytes-lit",
  "rand",
+ "rustc_version",
  "serde",
  "serde_json",
- "soroban-env-guest 22.0.0-rc.1.1",
- "soroban-env-host 22.0.0-rc.1.1",
- "soroban-ledger-snapshot 22.0.0-rc.1.1",
- "soroban-sdk-macros 22.0.0-rc.1.1",
+ "soroban-env-guest 22.0.0-rc.3",
+ "soroban-env-host 22.0.0-rc.3",
+ "soroban-ledger-snapshot 22.0.0-rc.3",
+ "soroban-sdk-macros 22.0.0-rc.3",
```

Comment:
@willemneal The fact versions are included for these soroban deps, means that somehow we're pulling in multiple versions of each. We should be only using one version of these across the whole repo, so some investigation is needed to figure out what is importing the old version and remove it.